test/mpi/maint/jenkins/xfail.conf

```diff
@@ -38,10 +38,6 @@
 # intermittently fail too (see issue 3770 for more details).
 * * * * * sed -i "s+\(^gather_big .*\)+\1 xfail=issue3770+g" test/mpi/coll/testlist
 
-################################################################################
-# Add lock option to tests that require large memory
-* * * * * sed -i "s+\(^large_type_sendrec .*\)+\1 lock=mem+g" test/mpi/datatype/testlist
```

Comment:
I presume we should just remove the commit that was added and then reverted before merging?

Reply:
Of course. I'll clean it up.